### PR TITLE
Correction of typos

### PR DESCRIPTION
**Pull request by XTRO123**

### CODE REVIEW

**Comment by LuckyCyborg** [comment]
Hi, Victor!

See https://developers.google.com/recaptcha/docs/display

Strange, but it is correct `grecaptcha`, not `recaptcha` in the rendering callback of Google ReCaptcha.

Then, please remove those changes. 

ALSO, I do not understand:
```
</a>$menuItems
```
What is with PHP variable directly into HTML code?

File: themes/AdminLite/Layouts/Backend.php [hunk 1]
```diff
@@ -102,7 +102,7 @@
       <!-- Sidebar toggle button-->
       <a href="#" class="sidebar-toggle" data-toggle="offcanvas" role="button">
         <span class="sr-only">Toggle navigation</span>
-      </a>
+      </a>$menuItems
```
Comment (LuckyCyborg) [inline themes/AdminLite/Layouts/Backend.php]: Why that?

File: modules/Users/Views/Registrar/Create.php [hunk 2]
```diff
@@ -69,7 +69,7 @@
 <script type="text/javascript">
 
 var captchaCallback = function() {
-    grecaptcha.render('captcha', {'sitekey' : '<?= Config::get("reCaptcha.siteKey"); ?>'});
+    recaptcha.render('captcha', {'sitekey' : '<?= Config::get("reCaptcha.siteKey"); ?>'});
```
Comment (LuckyCyborg) [inline modules/Users/Views/Registrar/Create.php]: Those should be kept as `grecaptcha` !

File: themes/AdminLite/Layouts/Backend.php (hunk 1 above)
Comment (XTRO123) [reply themes/AdminLite/Layouts/Backend.php]: Sorry, my mistake.

**Review by XTRO123 — COMMENTED** [review]
updated. Probably I need more time to sleep. ))

File: plugins/NestedSet/Database/ORM/Builder.php [hunk 3]
```diff
@@ -111,12 +111,9 @@ public function whereAncestorOf($id)
...
      * Get ancestors of specified node.
      *
-     * @since 2.0
-     *
-     * @param mixed $id
+     * @param $id
      * @param array $columns
-     *
-     * @return \Kalnoy\Nestedset\Collection
+     * @return \Nova\Database\ORM\Collection|static[]
```
Comment (LuckyCyborg) [inline plugins/NestedSet/Database/ORM/Builder.php]: That should be `Packages\NestedSet\Database\ORM\Collection`

File: plugins/NestedSet/Database/ORM/Builder.php [hunk 4]
```diff
@@ -213,12 +210,9 @@ public function orWhereNotDescendantOf($id)
...
      * Get descendants of specified node.
      *
-     * @since 2.0
-     *
-     * @param mixed $id
+     * @param $id
      * @param array $columns
-     *
-     * @return \Kalnoy\Nestedset\Collection
+     * @return \Nova\Database\ORM\Collection|Collection|static[]
```
Comment (LuckyCyborg) [inline plugins/NestedSet/Database/ORM/Builder.php]: That should be `Packages\NestedSet\Database\ORM\Collection`

**Comment by LuckyCyborg** [comment]
Thanks, Victor! 👍

**Pull request merged**